plugin-script/src/main/java/io/kestra/plugin/scripts/exec/scripts/runners/DockerScriptRunner.java

```diff
@@ -45,7 +45,7 @@
 
 public class DockerScriptRunner {
     private static final ReadableBytesTypeConverter READABLE_BYTES_TYPE_CONVERTER = new ReadableBytesTypeConverter();
-    public static final Pattern NEWLINE_PATTERN = Pattern.compile("[\\r\\n]+$");
+    public static final Pattern NEWLINE_PATTERN = Pattern.compile("([^\\r\\n]+)[\\r\\n]+");
```

Comment:
Isnt this one unnecessary as it catch the same as before, and the previous one is already annotated with `+` ? 
```suggestion
    public static final Pattern NEWLINE_PATTERN = Pattern.compile("([^\\r\\n]+)");
```

Reply:
nop this does not work we explicitely want the pattern to end with one of these char.
Like if the frame we get contains no newline it would match with your pattern but should not be sent:
```
Log("This log should not be flushed until we reach the newline or carry return here: \r")

onNext("his log should not be flushed until we reach ") < This would flush as it match the pattern without termination (basically it would match every frame at least once)
```

Reply:
we don't want to catch the end of the frame if it does not end with \r or \n. If there is a last line without line endin the oncomplete will flush it